package.json

```diff
@@ -59,12 +59,14 @@
     "enzyme": "^2.8.2",
     "eslint": "^3.17.1",
     "eslint-config-wordpress": "^1.1.0",
+    "eslint-plugin-jest": "20.0.3",
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-react": "^6.10.3",
     "expose-loader": "^0.7.3",
     "extract-text-webpack-plugin": "^2.1.0",
     "gettext-parser": "^1.2.2",
     "glob": "^7.1.1",
+    "jest": "20.0.4",
```

Comment:
It should be `~`, linters and for the most part rules will inherit the same _semantic behaviour_ from the linter, in this case ESLint and by proxy it's _shared plugins_ and _shared configs_:

> Via [ESLint Semantic Version Policy](https://github.com/eslint/eslint#semantic-versioning-policy):
> _"According to our policy, any minor update may report more errors than the previous release (ex: from a bug fix). As such, we recommend using the tilde (`~`) in `package.json` e.g. `"eslint": "~3.1.0"` to guarantee the results of your builds."_